OpenSim/Simulation/Model/PrescribedForce.cpp

```diff
@@ -74,6 +73,15 @@ PrescribedForce::PrescribedForce(SimTK::Xml::Element& aNode) : Super(aNode)
 void PrescribedForce::updateFromXMLNode(SimTK::Xml::Element& aNode, int versionNumber)
 {
     // Base class
+
```

Comment:
You still have to check that the version isn't 30505 (the current influx version, since you don't want to invoke this code block if deserializing files that have been converted during testing etc... even if it isn't a supported version.